jujugui/static/gui/src/app/assets/svgs/exposed_16.svg

```diff
@@ -0,0 +1,81 @@
+       transform="matrix(0.17638889,0,0,0.17638889,-27.919841,-415.17095)"
+       id="g6196">
+      <rect
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;fill:none;stroke:none;stroke-width:7.5;marker:none;enable-background:accumulate"
+         id="rect6186"
+         width="24"
+         height="24"
+         x="111.99999"
+         y="708.36218" />
+      <path
+         style="color:#000000;font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:medium;line-height:normal;font-family:sans-serif;font-variant-ligatures:normal;font-variant-position:normal;font-variant-caps:normal;font-variant-numeric:normal;font-variant-alternates:normal;font-feature-settings:normal;text-indent:0;text-align:start;text-decoration:none;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000000;letter-spacing:normal;word-spacing:normal;text-transform:none;writing-mode:lr-tb;direction:ltr;baseline-shift:baseline;text-anchor:start;white-space:normal;shape-padding:0;clip-rule:nonzero;display:inline;overflow:visible;visibility:visible;opacity:1;isolation:auto;mix-blend-mode:normal;color-interpolation:sRGB;color-interpolation-filters:linearRGB;solid-color:#000000;solid-opacity:1;fill:#808080;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:2.0111742;stroke-linecap:round;stroke-linejoin:round;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;marker:none;color-rendering:auto;image-rendering:auto;shape-rendering:auto;text-rendering:auto;enable-background:accumulate"
+         d="m 124,708.36133 c -6.61548,0 -12,5.38453 -12,12 0,6.61546 5.38452,12 12,12 6.61548,0 12,-5.38454 12,-12 0,-6.61547 -5.38452,-12 -12,-12 z m 0,2.01172 c 5.52856,0 9.98828,4.45974 9.98828,9.98828 0,5.52854 -4.45972,9.99023 -9.98828,9.99023 -5.52856,0 -9.98828,-4.46169 -9.98828,-9.99023 0,-5.52854 4.45972,-9.98828 9.98828,-9.98828 z"
+         id="ellipse6188"
+         inkscape:connector-curvature="0" />
+      <path
+         style="color:#000000;display:inline;overflow:visible;visibility:visible;opacity:1;fill:#808080;fill-opacity:1;fill-rule:nonzero;stroke:none;stroke-width:64;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;marker:none;enable-background:accumulate"
+         d="m 240,1384.6699 c -0.23819,0 -0.4761,0.017 -0.71289,0.043 L 239,1385 l -0.27148,-0.2168 c -2.21397,0.3907 -4.12616,1.7759 -5.1875,3.7578 L 235,1390 h 1 l 2,2 v -2 l 1,-1 h 1.99999 l 2,-2 v -2 c -0.7244,-0.2342 -2.23867,-0.3288 -2.99999,-0.3301 z M 238,1392 v 3 l 1,1 v 2 1 c 0.51741,0.1161 0.46973,0.328 1,0.3301 0.24933,0 0.49835,-0.019 0.74609,-0.047 l 0.2539,-0.2832 v -1 l 3,-3 1e-5,-1 -3,-2 z"
+         transform="matrix(1.5,0,0,1.5,-235.99999,-1367.6378)"
+         id="path4243"
+         inkscape:connector-curvature="0"
+         sodipodi:nodetypes="ccccccccccccccccccccccccccc" />
+    </g>
+  </g>
+</svg>
```

Comment:
Our new svg sprite system actually optimises these svgs when it builds the sprite, so no need to optimise initially!